ejs/scripts/services/validators_loader.ejs

```diff
@@ -1,20 +1,70 @@
 
 angular.module('validatorsApp').factory('ValidatorsLoader',['$http', function($http) {
 
-  var VALIDATOR_REGISTRY_API = '<%= VALIDATOR_REGISTRY_API %>'
+  var VALIDATOR_REGISTRY_API = '<%= VALIDATOR_REGISTRY_API %>';
+
+  function getValidations(callback) {
+      $http({
+        url: VALIDATOR_REGISTRY_API+"/validations",
+        method: "GET"
+      }).success(function(data, status, headers, config) {
+        callback(null, data.validations)
+      }).error(function(data, status, headers, config) {
+        callback(new Error(data))
+      })
+  }
+
+  function getValidators(callback) {
+      $http({
+        url: VALIDATOR_REGISTRY_API+"/validators",
+        method: "GET"
+      }).success(function(data, status, headers, config) {
+          callback(null, data.validators)
+      })
+      .error(function(data, status, headers, config) {
+          callback(new Error(data))
+      })
+  }
+
+  function reduceValidators(validators, validations) {
+    var reduced = []
+    validations.forEach(function(validation) {
+      var validator = _.find(validators, function(validator) {
+        return validator.validation_public_key === validation.validation_public_key
+      })
+      if (validator) {
+        validation.domain = validator.domain
+      }
+      reduced.push(validation)
+    })
+    return reduced
+  }
+
+  function sortByValidationsToday(validations) {
+    return _.sortBy(validations, function(validation) {
+      return validation.validations_count * -1
+    })
+  }
```

Comment:
Yes tests can be added, but since we don't have a testing strategy in place yet it will require a bit more configuration.